Mongo_Engine/Create/MongoServer.cs

```diff
@@ -53,15 +58,17 @@ public MongoServer(string folderName)
 
                 Thread.Sleep(1000);
                 m_Client = new MongoClient(@"mongodb://localhost:27017");
+                alert = "MongoDB Files Stored in " + folderName;
             }
+            return alert;
         }
```

Comment:
I'd reformat beside just porting. I am not sure why we kept the fields at this point. If you're calling the `mongod` command, your mongo is added to the path already, so when you start it you should be able to get the dbpath with something like `client.GetDatabase("admin").RunCommand<BsonDocument>("{getCmdLineOpts: 1}")["parsed"]["storage"]["dbPath"].ToString()`